draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -171,7 +171,11 @@ specify just the year. -->
     the saved_cwnd for the path and the minimum Round Trip Time (RTT). These
     parameters are saved and used to modify the CC
     behavior of a subsequent connection between the
-    same endpoints.</t>
+    same endpoints. Some congestion control algorithms may use other parameters.
+    For example, implementations using BBR will want to also retain the
+    value of the sending rate required to reach the full bandwidth available to
```

Comment:
What is /full/ correct here? or ought just say /bandwidth/ ... which might be the word to match use for bbr, or /capacity/ if we wish to be consistent?

Reply:
Yiu are right, this is loose language. BBR has the concepts of "bottleneck bandwidth" (the control variable), "pacing rate" (a multiple of the bottleneck bandwidth) and "delivery rate" (the value measured from the stream of ACKs). I will rewrite.